client/src/pages/dashboard/components/Nhl/TeamStats/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from "react";
+import { string } from "prop-types";
+import axios from "axios";
+import emoji from "emoji-dictionary";
+import styled from "styled-components";
+import theme from "../../../../../style/theme";
+
+const DivFlex = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const Head4 = styled.h4`
+  width: 100%;
+  margin-left: 40px;
+`;
+
+const TeamStats = ({ teamId }) => {
+  const [regSeason, setRegSeason] = useState("");
+  const [postSeason, setPostRegSeason] = useState("");
+
+  useEffect(() => {
+    axios
+      .get(`https://statsapi.web.nhl.com/api/v1/teams/${teamId}/stats`)
+      .then((data) => {
+        setRegSeason(data.data.stats[0].splits[0].stat);
+        setPostRegSeason(data.data.stats[0].type.gameType.postseason);
+      });
+  }, []);
+
+  console.log("regSeason", regSeason);
```

Comment:
I think that it should be deleted.

Reply:
https://github.com/dimapanasiuk/mern/commit/236eaf5e0df4c1fd11db9908fd397c138376745c